src/Layout/PresentationMenu/PresentationSelection/PresentationSelection.tsx

```diff
@@ -28,13 +28,61 @@ export class PresentationSelection extends React.Component<PresentationSelection
         });
     };
 
+    selectTooltipContent(table: string | undefined) {
+        switch (table) {
+            case "TABLE_ELECTION_OVERVIEW": {
+                return {
+                    text: "Her kan du se valgresultatet på landsbasis. Trykk på meg for mer informasjon!",
```

Comment:
I would suggest we remove the "Trykk på meg for mer informasjon!" as this is already mentioned in the tutorial.